### PR TITLE
Semantic HTML Refactoring - Marketing Agency

### PR DESCRIPTION
**Pull request by conorjkelly96**

### CODE REVIEW

**Comment by conorjkelly96** [comment]
### Work Done
- Refactored HTML code to include semantic elements instead of standard / generic elements
- Refactored CSS code to reduce duplication of styling blocks
- Restructured HTML class elements to allow for more concise CSS code

### TODO
- [ ] Would like to add a media query to this to ensure the page was accessible to different mobile viewports.

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+.navbar h1 {
+  display: inline-block;
+  font-size: 48px;
+}
+
+.navbar h1 .seo {
+  color: #d9dcd6;
+}
+
```
Comment (LeeFarnell2U) [inline assets/css/style.css:44]: This does work, but I would put the `h1` styling before the `ul`/`li` as the `h1` comes first in the navbar.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+    <!-- main content section about key services offered to clients -->
+    <main class="content">
+      <section class="content-box" id="search-engine-optimization">
+        <img
+          src="./assets/images/search-engine-optimization.jpg"
+          class="float-left"
+          alt="On top of a wooden table, there is a notepad, pencils a laptop with a magnifying glass and some coffee"
+        />
+
```
Comment (LeeFarnell2U) [inline index.html:47]: Some of your `alt` tags are quite long but they don't have to be. You could shorten them but there is nothing wrong with this.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+        <p>
+          As the search ranking for your business increases, your advertising
+          costs decrease, and you no longer need to advertise your page.
+        </p>
+      </section>
+    </aside>
+
+    <!-- Footer section containing company information -->
+    <footer class="footer-wrapper">
+      <h4>Made with ❤️️ by Horiseon</h4>
```
Comment (LeeFarnell2U) [inline index.html:129]: As this is a footer, this doesn't need to be a `h4` as it's in a different section, but I like that you followed the correct structure for your `h` tags.

Comment (conorjkelly96) [reply index.html:129]: Understood!

File: index.html (hunk 2 above)
Comment (conorjkelly96) [reply index.html:47]: Did think it was quite long, thanks for the advice!

**Pull request merged**